projects/entry-form/src/styles/modules/components/forms/checkboxs/_general.scss

```diff
@@ -21,10 +21,26 @@
             stroke: map.value-from($theme, 'checkbox.checkmark-color')!important;
         }
 
+        &.mat-checkbox-checked .mat-checkbox-background .mat-checkbox-checkmark-path {}
+
         &:hover {
             .mat-checkbox-frame {
                 border: map.value-from($theme, 'checkbox.border-hover');
             }
         }
     }
+
+    &.mat-checkbox-checked {
```

Comment:
Not sure why form needs this all of this suddenly?

Reply:
if we want to have selected state then yes